### PR TITLE
Update expandable icons to SVG

### PR DESCRIPTION
**Pull request by anselmbradford**
Spins SVG icon update in privacy act statement secondary navigation expandables from https://github.com/cfpb/cfgov-refresh/pull/3887

## Changes

- Updates expandables in privacy act statement secondary navigation https://github.com/cfpb/cfgov-refresh/pull/3887
- Fixes issue where privacy act statement on /fin-ed/privacy-act-statement/ erroneously had markup that said the expandable was initialized while there was no expandable JS attached to that page.

## Testing

1. Visit /fin-ed/privacy-act-statement/ and resize to mobile size.
2. Inspect the expandable and see that it's an SVG.
3. Do the same for /privacy/

## Screenshots

screen shot 2018-05-16 at 11 59 42 am

### CODE REVIEW

**Comment by Scotchester** [comment]
May I suggest a renaming of this PR? The primary effect of this PR is to replace **all** expandable cue icons.

File: cfgov/unprocessed/js/routes/fin-ed/privacy-act-statement/index.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/* ==========================================================================
+   Scripts for `/fin-ed/privacy-act-statement/.
+   ========================================================================== */
+
+const atomicHelpers = require( '../../../modules/util/atomic-helpers' );
+const Expandable = require( '../../../organisms/Expandable' );
+console.log( document.querySelector(
```
Comment (jimmynotjim) [inline cfgov/unprocessed/js/routes/fin-ed/privacy-act-statement/index.js]: Remove debugging code.

**Comment by jimmynotjim** [comment]
If this is going to be all expandables, please include these other changes as well 
- https://github.com/cfpb/cfgov-refresh/pull/3887/files#diff-0213fe87f560c06b097a0130ea2f3eaa 
- https://github.com/cfpb/cfgov-refresh/pull/3887/files#diff-f1481609f3f008cf2274aec7455b1e10 
- https://github.com/cfpb/cfgov-refresh/pull/3887/files#diff-ea7aba4c4e268a13b784481eca86405d
- https://github.com/cfpb/cfgov-refresh/pull/3887/files#diff-13377cbaabf59c01cfdebe7d4f3e2a1d
- https://github.com/cfpb/cfgov-refresh/pull/3887/files#diff-3dc41300db3bf86bdb798ceebecab233
- test/unit_tests/apps/financial-well-being/js/fwb-results-spec.js (I've actually been removing the icons from the test fixtures because we don't seem to ever check them so I don't know if it's necessary to include them here)

File: cfgov/unprocessed/js/routes/fin-ed/privacy-act-statement/index.js (hunk 1 above)
Comment (anselmbradford) [reply cfgov/unprocessed/js/routes/fin-ed/privacy-act-statement/index.js]: This is some kind of karma! 😆

Comment (anselmbradford) [reply cfgov/unprocessed/js/routes/fin-ed/privacy-act-statement/index.js]: Fixed

File: cfgov/jinja2/v1/job-description-page/index.html [hunk 2]
```diff
@@ -120,11 +120,11 @@ <h4>Position type</h4>
             <a class="o-expandable_target o-expandable_link" title="Expand content">
                 <span class="o-expandable_cue o-expandable_cue-open">
                     Read full description
-                    <span class="cf-icon cf-icon-plus-round"></span>
+                    {{ svg_icon('plus-round') }}
```
[the remark this answers is not in the document]
Comment (anselmbradford) [reply cfgov/jinja2/v1/job-description-page/index.html]: Fixed!

[the remark this answers is not in the document]
Comment (jimmynotjim) [reply cfgov/jinja2/v1/job-description-page/index.html]: Thanks, sorry to lead you astray...twice 😆

**Pull request merged**